src/test/java/com/javastudy/coworkings/dao/jdbc/JdbcCoworkingDaoTest.java

```diff
@@ -4,18 +4,103 @@
 import com.javastudy.coworkings.dao.CoworkingDao;
 
 import com.javastudy.coworkings.entity.Coworking;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.sql.SQLException;
 import java.util.List;
 
+import static com.javastudy.coworkings.util.DataSourceLoader.*;
 import static org.junit.jupiter.api.Assertions.assertEquals;
 
 public class JdbcCoworkingDaoTest {
+    private CoworkingDao coworkingDao = ServiceLocator.getService(CoworkingDao.class);
+    private static Coworking actualCoworking;
+
+    @BeforeAll
+    static void before() throws IOException, URISyntaxException {
+        init("db/dml_db_data.sql");
+        ServiceLocator.register(CoworkingDao.class, new JdbcCoworkingDao(getDataSource()));
+        actualCoworking = Coworking.builder()
+                .id((long) 1)
+                .name ("TestName")
+                .mainImage ("http://testlink")
+                .overview ("TestOverview")
+                .location ("TestLocation")
+                .reviewsCount ((long)1)
+                .city ("Kiev")
+                .dayPrice (1)
+                .weekPrice (2)
+                .monthPrice (3)
+                .rating (5)
+                .openingHours ("TestOpeningHours")
+                .containsDesk (true)
+                .containsOffice (true)
+                .containsMeetingRoom (true)
+                .build();
+    }
+
+    @BeforeEach
+    void setUp() throws SQLException {
+        reloadData();
+    }
+
+    @AfterEach
+    void tearDown() throws SQLException {
+        cleanSchema();
+    }
 
     @Test
     public void testSearchByName() {
-        CoworkingDao coworkingDao = ServiceLocator.getService(JdbcCoworkingDao.class);
-        List<Coworking> listOfCoworkings = coworkingDao.searchByName("Crea");
-        assertEquals(4, listOfCoworkings.size());
+
+        List<Coworking> factCoworking = coworkingDao.searchByName("test");
+
+        assertEquals(1, factCoworking.size());
+        assertEquals(actualCoworking.getId(), factCoworking.get(0).getId());
```

Comment:
actualCoworking -> should be renamed to expectedCoworking
Coworking actualCoworking = factCoworking.get(0); // fact? no shortcuts please. Use f@#k instead.
 assertEquals(expectedCoworking.getName(), actualCoworking.getName());